sidebars.js

```diff
@@ -461,6 +473,67 @@ const sidebars = {
         },
       ],
     },
+    {
+      type: "category",
+      label: "Applications",
+      collapsible: true,
+      collapsed: true,
+      items: [
+        {
+          type: "category",
+          label: "Warp Routes",
```

Comment:
done https://github.com/hyperlane-xyz/v3-docs/pull/216/commits/8d4d9a06ee3e0d78e8193b919ab7f1faf2ca3bb9

i like the consistency of having them all collapsed, but agree that in this case it makes sense to edit so that people can directly jump to warp routes when they come on the docs.